tests/units/typing/tensor/test_torch_tensor.py

```diff
@@ -177,3 +177,16 @@ class MMdoc(BaseDoc):
 
     doc_copy.embedding = torch.randn(32)
     assert not (doc.embedding == doc_copy.embedding).all()
+
+
+def test_serialization():
+    from docarray import BaseDoc
+
+    class MyDoc(BaseDoc):
+        tens: TorchTensor
+
+    doc = MyDoc(tens=torch.rand(10, requires_grad=False))
```

Comment:
can we please use `parametrization`? Like this, when a test fails is easier to identify which failed.

Also, can u do some sort of assertion on the output of this `json` method?